src/storage/BaseProcessor.inl

```diff
@@ -146,11 +154,21 @@ void BaseProcessor<RESP>::collectProps(RowReader* reader,
         if (reader != nullptr) {
             const auto& name = prop.prop_.get_name();
             auto res = RowReader::getPropByName(reader, name);
+            VariantType v;
             if (!ok(res)) {
-                VLOG(1) << "Skip the bad value for prop " << name;
-                continue;
+                VLOG(1) << "Bad value for prop: " << name;
+                // TODO: Should return NULL
+                auto defaultVal = RowReader::getDefaultProp(prop.type_.type);
+                if (!defaultVal.ok()) {
+                    // Should never reach here.
+                    LOG(FATAL) << "Get default value failed for " << name;
```

Comment:
If the name not found in schema and the default value is not defined ?

Reply:
> If the name not found in schema and the default value is not defined ?

Emm.. have not using schema at here, it simply returns A default value based on the type, not the default value of the schema itself.